bin/cli.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
```

Comment:
I don't see the mode of this file. Is it executable? If not, can you please `chmod` it to be?